Network/HTTP/Req.hs

```diff
@@ -3,18 +3,19 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE DeriveLift #-}
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE KindSignatures #-}
```

Comment:
Went through all the extensions I thought I added and tested if they were necessary. This wasn't, but I also actually didn't add it, so it's strictly speaking unrelated to the rest of the PR.